cli/js/workers_test.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+import { test, assert, assertEquals } from "./test_util.ts";
+import { runIfMain } from "../../std/testing/mod.ts";
```

Comment:
I don't think we should import from std ... afaik this is not done in cli/js.. You can get runIfMain from cli/js/test_util.ts